src/DIRAC/DataManagementSystem/scripts/dirac_dms_directory_sync.py

```diff
@@ -232,21 +220,21 @@ def uploadLocalFile(dm, lfn, localfile, storage):
         """
         Upload a local file to a storage element
         """
-        res = dm.putAndRegister(lfn, localfile, storage, None)
+        res = returnSingleResult(dm.putAndRegister(lfn, localfile, storage, None))
```

Comment:
can't that whole `uploadLocalFile` be replaced with `returnSingleResult(dm.putAndRegister(lfn, localfile, storage, None))` ?

Reply:
Yes, it can be replaced like that

Reply:
Replaced this function, and downloadFile function